### PR TITLE
add flag to disable startup setlocal indent

### PR DESCRIPTION
**Pull request by aykamko**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -74,6 +74,11 @@ With content already, it will be aligned to the opening parenthesis::
 
 Existing indentation (including ``0``) in multiline strings will be kept, so this setting only applies to the indentation of new/empty lines.
 
+g:python_pep8_setlocal_indent
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, this plugin sets ``shiftwidth``, ``tabstop``, and ``softtabstop`` to 4 on startup. This behavior can be disabled adding ``let g:python_pep8_setlocal_indent = 0`` to your vimrc.
+
```
Comment (blueyed) [inline README.rst]: What about using the value from there?

Comment (blueyed) [inline README.rst]: (4 being the default then)

Comment (aldanor) [inline README.rst]: I guess the point is for working with multiple repos, e.g. TensorFlow uses sw=2

Comment (blueyed) [inline README.rst]: Yes.  `0` would still disable it (in case you want to set it yourself), but you could specify 2 here then for example.

**Comment by blueyed** [comment]
Could this be combined with https://github.com/Vimjas/vim-python-pep8-indent/pull/79 ?

**Comment by blueyed** [comment]
Related: https://github.com/Vimjas/vim-python-pep8-indent/pull/70

Maybe we should just remove all 3 of them?
It is not the job of an indent plugin to do this, but rather of a ftplugin.

**Pull request closed without merging**